common/normalize.go

```diff
@@ -0,0 +1,32 @@
+package common
+
+import (
+	"unicode"
+
+	"github.com/mtibben/confusables"
+	"golang.org/x/text/transform"
+	"golang.org/x/text/unicode/norm"
+)
+
+func isMn(r rune) bool {
+	return unicode.Is(unicode.Mn, r)
+}
+
+// stupid name, cant think of a better one atm
+func FixText(content string, diacritics, matchConfusables bool) string {
```

Comment:
Given how you're already having a `matchConfusables`, also consider this change:
```suggestion
func FixText(content string, removeDiacritics, matchConfusables bool) string {
```
(Or something similar to that extent)

Reply:
Agreed, more consistent that way